src/Spout/Common/Entity/Style/Style.php

```diff
@@ -490,13 +490,13 @@ public function isRegistered() : bool
         return $this->isRegistered;
     }
 
-    public function register(int $id) : void
+    public function markAsRegistered(?int $id) : void
```

Comment:
can this take a `null`?

Reply:
Yes, Style id from StyleBuilder is null and it can be passed in construct of `StyleRegistry` for example